py-polars/tests/unit/operations/map/test_map_elements.py

```diff
@@ -295,6 +295,24 @@ def test_map_elements_on_empty_col_10639() -> None:
     }
 
 
+@pytest.mark.filterwarnings(
+    "ignore:.*("
+    "This process .* is multi-threaded|"
+    "ast.Num is deprecated|"
+    "Attribute n is deprecated"
+    ").*:DeprecationWarning"
```

Comment:
these are only raised on 3.12, will report to joblib and check they're ok

Reply:
the first one seems legit - I'll take a closer look, will mark for review when ready